crates/bevy_ecs/src/world/mod.rs

```diff
@@ -189,6 +201,26 @@ impl World {
             .init_component_with_descriptor(&mut self.storages, descriptor)
     }
 
+    /// Retrieves the [`ComponentId`] of the given [`Component`] type in
+    /// this [`World`]. Returns [`None`] if the [`Component`] type has not
```

Comment:
Empty line after the first sentence. It helps seperate the at-a-glance info from the details nicely.